packages/itwin/grouping-mapping-widget/src/components/Mappings/hooks/useFetchExtractionStatus.ts

```diff
@@ -23,16 +24,16 @@ export const useFetchExtractionStatus = ({
     placeholderData: undefined,
     queryFn: async () => {
       const accessToken = await getAccessToken();
-      const extraction = extractionClient.getExtractionHistoryIterator(accessToken, iModelId, 1);
+      const extraction = extractionClient.getIModelExtractionsIterator(accessToken, iModelId, 1);
       const latestExtractionResult = await extraction.next();
 
       let extractionStatusIcon: ExtractionStatusData;
       let extractionMessageData: ExtractionMessageData[] = [];
 
-      const jobId = latestExtractionResult.value?.jobId;
+      const latestExtractionResultId = latestExtractionResult.value?.id;
       let latestJobStatus: ExtractionStatus|undefined;
-      if (jobId) {
-        latestJobStatus = await extractionClient.getExtractionStatus(accessToken, jobId);
+      if (latestExtractionResultId) {
+        latestJobStatus = await extractionClient.getExtractionStatus(accessToken, latestExtractionResultId);
```

Comment:
There should be no need to use `getExtractionStatus` after `getting the extraction from `getIModelExtractionsIterator` since that extraction entry will already have a `state` property on it.

Reply:
Removed and used the state field of the `latestExtractionResult`.

Reply:
Resolved.